zeebe/broker-client/src/main/java/io/camunda/zeebe/broker/client/impl/BrokerTopologyManagerImpl.java

```diff
@@ -241,4 +244,19 @@ public void onClusterConfigurationUpdated(final ClusterConfiguration clusterTopo
           }
         });
   }
+
+  private void notifyOnClusterChangeCompleted(
+      final ClusterConfiguration clusterTopology, final BrokerClusterStateImpl topologyToUpdate) {
+
+    clusterTopology
+        .lastChange()
+        .filter(lastChange -> lastChange.id() > topologyToUpdate.getLastCompletedChangeId())
+        .ifPresent(
+            lastChange -> {
+              LOG.info("The cluster has for sure been updated with new id " + lastChange.id());
```

Comment:
:x: That's not something the user needs to see, please lower to debug level.

:wrench: I'd clarify the message to something like "Updating topology with last completed change {}`.